sycl/doc/extensions/C-CXX-StandardLibrary/DeviceLibExtensions.rst

```diff
@@ -32,3 +32,158 @@ Arguments:
+   double __devicelib_cbrt(double x);
+   float  __devicelib_cbrtf(float x);
+   double __devicelib_hypot(double x, double y);
+   float  __devicelib_hypotf(float x, float y);
+   double __devicelib_erf(double x);
+   float  __devicelib_erff(float x);
+   double __devicelib_erfc(double x);
+   float  __devicelib_erfcf(float x);
+   double __devicelib_tgamma(double x);
+   float  __devicelib_tgammaf(float x);
+   double __devicelib_lgamma(double x);
+   float  __devicelib_lgammaf(float x);
+   double __devicelib_fmod(double x, double y);
+   float  __devicelib_fmodf(float x, float y);
+   double __devicelib_remainder(double x, double y);
+   float  __devicelib_remainderf(float x, float y);
+   double __devicelib_remquo(double x, double y, int *q);
+   float  __devicelib_remquof(float x, float y, int *q);
+   double __devicelib_nextafter(double x, double y);
+   float  __devicelib_nextafterf(float x, float y);
+   double __devicelib_fdim(double x, double y);
+   float  __devicelib_fdimf(float x, float y);
+   double __devicelib_fma(double x, double y, double z);
+   float  __devicelib_fmaf(float x, float y, float z);
+   float  __devicelib_asinf(float x);
+   double __devicelib_asin(double x);
+   float  __devicelib_atanf(float x);
+   double __devicelib_atan(double x);
+   float  __devicelib_atan2f(float x, float y);
+   double __devicelib_atan2(double x, double y);
+   float  __devicelib_coshf(float x);
+   double __devicelib_cosh(double x);
+   float  __devicelib_sinhf(float x);
+   double __devicelib_sinh(double x);
+   float  __devicelib_tanhf(float x);
+   double __devicelib_tanh(double x);
+   float  __devicelib_acoshf(float x);
+   double __devicelib_acosh(double x);
+   float  __devicelib_asinhf(float x);
+   double __devicelib_asinh(double x);
+   float  __devicelib_atanhf(float x);
+   double __devicelib_atanh(double x);
+   float  __devicelib_frexpf(float x, int *exp);
+   double __devicelib_frexp(double x, int *exp);
+   float  __devicelib_ldexpf(float x, int exp);
+   double __devicelib_ldexp(double x, int exp);
+   float  __devicelib_log10f(float x);
+   double __devicelib_log10(double x);
+   float  __devicelib_modff(float x, float *intpart);
+   double __devicelib_modf(double x, double *intpart);
+   float  __devicelib_expf(float x);
+   double __devicelib_exp(double x);
+   float  __devicelib_exp2f(float x);
+   double __devicelib_exp2(double x);
+   float  __devicelib_expm1f(float x);
+   double __devicelib_expm1(double x);
+   int    __devicelib_ilogbf(float x);
+   int    __devicelib_ilogb(double x);
+   float  __devicelib_log1pf(float x);
+   double __devicelib_log1p(double x);
+   float  __devicelib_log2f(float x);
+   double __devicelib_log2(double x);
+   float  __devicelib_logbf(float x);
+   double __devicelib_logb(double x);
+
+Semantic:
+Those __devicelib_* functions have the same semantic as corresponding math functions
+from <math.h> or <cmath>. Please refer to: http://www.cplusplus.com/reference/cmath/
```

Comment:
Please, replace all references to web-sites citing C++ spec with direct references to the spec.